src/taskparameters.ts

```diff
@@ -67,12 +66,14 @@ export class TaskParameters {
             this._gpuSKU = (gpuSku == 'K80') ? 'K80' : ( gpuSku == 'P100' ? 'P100' : 'V100');
         }
         this._image = core.getInput('image', { required: true });
+
         let ipAddress = core.getInput('ip-address');
-        if(ipAddress != "Public" && "Private") {
+        if(["Public", "Private"].includes(ipAddress) == false) {
```

Comment:
The linters I use on my other projects would flag this and the similar lines due to using `==` instead of `===` - not that there would be any difference in this case.

Reply:
Thanks @jesseward, @kf6kjg for your inputs :)
@jesseward, it will be great if you can update this PR with the above small change? 
Then, we can take this PR forward.